frontend/pulse3d/src/utils/generic.js

```diff
@@ -60,17 +60,19 @@ const getPeaksValleysFromTable = async (table) => {
   const wellNames = new Set(columns.map((name) => name.split("__")[0]));
 
   // filter out null values (0) and some values get randomly parsed to bigint values which cannot be converted to JSON
-  const columnData = table.data[0].children.map(({ values }) =>
-    Array.from(values)
+  const columnData = table.data[0].children.map(({ values }) => {
```

Comment:
@luciipak are any of the changes in this file needed? For some reason they didn't show up in the conflicts when I merged main